gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/model/mapper/ApiMigration.java

```diff
@@ -156,6 +160,29 @@ private Endpoint mapEndpoint(io.gravitee.definition.model.Endpoint lb) {
             .build();
     }
 
+    private Analytics mapAnalytics(Logging v2logging) {
+        var loggingEnabled = isLoggingEnabled(v2logging);
+        var v4logging = loggingEnabled ? mapLogging(v2logging) : new io.gravitee.definition.model.v4.analytics.logging.Logging();
+        return Analytics.builder().enabled(loggingEnabled).logging(v4logging).build();
+    }
+
+    private boolean isLoggingEnabled(Logging v2logging) {
+        if (v2logging == null) {
+            return false;
+        }
+        return !v2logging.getMode().equals(io.gravitee.definition.model.LoggingMode.NONE);
```

Comment:
for enum you don’t need `equals()`

```suggestion
        return v2logging != null && v2logging.getMode() != io.gravitee.definition.model.LoggingMode.NONE;
```